src/main/java/newbank/server/Account.java

```diff
@@ -30,6 +31,19 @@ private void validate(String accountName, Money openingBalance) throws AccountNa
   }
 
   public String toString() {
-    return String.format("%s: %.2f %s", accountName, openingBalance.getNumberStripped(), openingBalance.getCurrency());
+    return String.format(
+        "%s: %.2f %s", accountName, balance.getNumberStripped(), balance.getCurrency());
+  }
+
+  public String getName() {
+    return accountName;
+  }
+
+  public Money getBalance() {
+    return balance;
+  }
+
+  public void addMoney(Money amount) {
+    balance = balance.add(Money.of(amount.getNumber(), "GBP"));
```

Comment:
Shall we extract `"GBP"` to a constant in the class?

Reply:
Done.